tests/test/paywall-erc20.test.js

```diff
@@ -0,0 +1,89 @@
+const url = require('../helpers/url')
+const dashboard = require('../helpers/dashboard')
+const wait = require('../helpers/wait')
+// const debug = require('../helpers/debugging')
+
+// const it = debug.screenshotOnFail(page)
+
+const lockName = 'A Lock For the Paywall'
+const lockKeyPrice = '1.00'
+
+let lockSelector
+
+describe('The Unlock Paywall (ERC20 lock)', () => {
+  beforeAll(async () => {
+    // We first need to deploy a new lock
+    const lock = await dashboard.deployLock(
```

Comment:
Is that really needed? I am worried our integration tests are already the bottleneck? Maybe we can deploy a lock as part of the setup process?

Reply:
@akeem has the same concern (see #4310). In response, I made this: #4314.

It creates all the locks needed in the standup, so they would also exist for regular dev work.

I didn't mention this in the PR because I didn't have hard evidence to document the difference, but it does feel like the tests run a lot faster with #4314.